### PR TITLE
Add cbr config option for opus

### PR DESCRIPTION
**Pull request by GEverding**
Setup wiring to accept `opusCbr` for configuring opus.

### CODE REVIEW

File: src/ortc.cpp [hunk 1]
```diff
@@ -920,6 +920,7 @@ namespace mediasoupclient
 			auto opusMaxPlaybackRateIt     = params.find("opusMaxPlaybackRate");
 			auto opusMaxAverageBitrateIt   = params.find("opusMaxAverageBitrate");
 			auto opusPtimeIt               = params.find("opusPtime");
+			auto opusCbrIt                 = params.find("opusCbr");
```
Comment (jmillan) [inline src/ortc.cpp]: Can you please put the `cbr` logic after  the `dtx` one among the PR? They belong to the same logical group of options.

Thanks.

Comment (GEverding) [reply src/ortc.cpp]: @jmillan moved.

**Review by jmillan — APPROVED** [review]

**Comment by jmillan** [comment]
Thanks @GEverding!

**Comment by GEverding** [comment]
Can I get this merged in? I'm trying to clean some branches to fix some other bugs

**Review by ibc — APPROVED** [review]

**Pull request merged**